sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -806,6 +807,66 @@ aditof::Status CameraItof::requestFrame(aditof::Frame *frame,
     return Status::OK;
 }
 
+aditof::Status CameraItof::normalizeABdata(aditof::Frame *frame,
+                                           int normalizeSteps,
+                                           uint16_t **abData) {
+
+    using namespace aditof;
+    Status status = Status::OK;
+
+    if (normalizeSteps != 1 && normalizeSteps != 2) {
+        return Status::INVALID_ARGUMENT;
+    }
+
+    status = frame->getData("ab", abData);
```

Comment:
There might be some configurations where AB is nor avaiable. It would make sense to check the status that getData returns and return early if there's an error. Otherwise, invalid memory will be accessed.